SU-SWS/stanford_profile
# D8CORE-5221 D8CORE-4857 D8CORE-3517 Small adjustments to configs and content.

# READY FOR REVIEW

# Summary
- D8CORE-5221 Space before external link text
- D8CORE-4857 Adjust media permissions
- D8CORE-3517 Add a placeholder image


# Need Review By (Date)
- 1/20

# Urgency
- medium

# Steps to Test
1. Checkout this branch
2. do a fresh install
3. go to `/admin/media`
4. validate a "Placeholder" image exists
5. Create a page with an external link
6. Verify the external link has a space before the open parenthesis: `<title> (link is external)</title>`.
7. Review permissions to see a contributor can edit an image.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
